src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -76,6 +84,17 @@ public void NotifyInitializationCompleted()
 
         public void Dispose()
         {
+            if (!_isTelemetryEmitted)
+            {
+                // Work around to detect if PMC loaded automatically because it was last focused window.
+                bool? reopenAtStart = IsLoaded;
```

Comment:
So reading up more on this (and testing myself), here's when Loaded is true.

Loaded is true only when the powershell window is focusing at the time of closing VS.
Loaded is false in every other occassion.

Say I closed my solution with PMC as the last focused window.
I reopen VS, then for that session, solution load is impacted by the powershell host loading.
At the end of that session, if PMC is not the focused window anymore we won't account for the fact this session had loaded the powershell host unintentionally.

Reply:
@nkolev92 

Then that next VS reopen is separate telemetry. When you open reopen VS it'll record PMC is loaded, since it's not focused anymore then reOpenAtStart would be false for this new telemetry session.

Reply:
Also I just changed logic here. Can you check?